operations/iuf/IUF.md

```diff
@@ -675,3 +676,11 @@ If an error is encountered while executing `iuf run`, `iuf` will attempt to comp
   - Remove the undesired product from the IUF media directory.
   - Add the new release of the product to the IUF media directory.
   - Execute `iuf run` using a **new** activity identifier but otherwise use the same arguments specified previously in order to re-run the install or upgrade process.
+
+## Install and Upgrade Observability Framework
+
+The Install and Upgrade Observability Framework creates unified consistent requirements for each product including assertions for Goss health checks, as well as metrics and dashboards for health checks. The framework also includes a unified consistent method
```

Comment:
I don't understand what "unified consistent requirements" means.  Can this be reworded?

Reply:
Made those changes